pom.xml

```diff
@@ -67,7 +67,11 @@
       <artifactId>kubernetes-client</artifactId>
       <version>${kubernetes-client.version}</version>
     </dependency>
-
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>openshift-client</artifactId>
```

Comment:
No, we don't I was trying something out and forgot to remove that in the end. Let me remove that.